src/index.ts

```diff
@@ -75,4 +89,37 @@ export class ContentfulLivePreview {
       [TagAttributes.LOCALE]: locale,
     };
   }
+
+  static togglefieldTagging(): boolean {
+    return (this.fieldTaggingEnabled = !this.fieldTaggingEnabled);
+  }
+
+  static toggleLiveUpdatesMode(): boolean {
+    return (this.liveUpdatesEnabled = !this.liveUpdatesEnabled);
+  }
+
+  static setupFieldTagging(): void {
+    if (this.fieldTaggingEnabled) {
+      ContentfulLivePreview.fieldTagging = new FieldTagging();
+      window.addEventListener('message', (event) => {
+        if (typeof event.data !== 'object' || !event.data) return;
```

Comment:
can we maybe then extract this checks if we bind two event listeners?
something like `validateMessage`

or keep one event listener 🤔